index.html

```diff
@@ -136,6 +136,25 @@ <h3 class="aboutmesub">Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="forms">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in </h2>
+            <form action="https://formspree.io/f/form_id" method="post">
+                <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
+                <input type="email" name="email" id="email" placeholder="Email address" required>
+                <textarea name="msg" id="msg" placeholder="Enter text here" required maxlength="500"></textarea>
+                <button type="submit">Get in touch</button>
```

Comment:
- [ ] Button interactions such as hover, and active (when it's pressed), are styled as shown in the Figma template, kindly implement this so the buttons have some interactivity 

![Screenshot_20220705_192332](https://user-images.githubusercontent.com/25227022/177392078-5f3c863e-b839-4eb9-960c-22f9346b04ba.png)